pom.xml

```diff
@@ -0,0 +1,37 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.debortoliwines</groupId>
+  <artifactId>odoo-java-api</artifactId>
+  <version>1.5.0</version>
```

Comment:
Hi @fpoyer ,

Please, change the release here and bump it to 2.0.0 so that the backward compatibilty will be clear in the releasing.

regards